test/testcafe/framework/page-objects/YubiKeyAuthenticatorPageObject.js

```diff
@@ -9,7 +10,22 @@ export default class YubiKeyAuthenticatorPageObject extends BasePageObject {
     return this.form.setTextBoxValue(name, value);
   }
 
-  submit() {
-    return this.form.clickSaveButton();
+  async waitForYubikeyView() {
+    await this.form.el.find('form').exists;
+  }
+
+  clickEnrollButton() {
+    if(userVariables.v3) {
+      return this.form.clickButton('Set up');
+    }
+    return this.clickVerifyButton();
+  }
+
+  clickVerifyButton() {
+    return this.form.clickButton('Verify');
+  }
```

Comment:
Just checking: this replaces `submit()` because the semantic selector needs to find by text, right?

Reply:
thats right it needs to find the button semantically by text.  Especially since in V3 the button text between the enroll and verify flow are different.